packages/UsageAnalysis/files/Test Track/Queries/MS SQL.md

```diff
@@ -0,0 +1,35 @@
+### Adding
```

Comment:
Why is it one file instead of 4 test cases?

Reply:
this is the additional test. PostgresDart testing is separated.
For MySQL I decided to put them together because they are quick, testing same functionality but for MySQL and will likely to be deleted after some time (will be autotested)